src/_data/how_to/get_connected.yml

```diff
@@ -40,7 +40,7 @@ toc:
-blue border-bottom-0 border-top-0 border-left-0 border-right-0"><em>3k - 9,999 SIMs</em></td> <td class="border-dark-blue border-bottom-0 border-top-0 border-left-0"><em>>9,999 SIMs</em></td> <td class="border-dark-blue border-bottom-0 border-top-0">-</td> <td class="border-dark-blue border-bottom-0 border-top-0">-</td> <td class="border-dark-blue border-bottom-0 border-top-0">-</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">1 GB</th> <td class="border-dark-blue border-left-0 border-right-0">$10.00</td> <td class="border-dark-blue border-left-0 border-right-0">$7.50</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$4.99</td> <td class="border-dark-blue table-bordered">$20.00</td> <td class="border-dark-blue table-bordered">$8.00</td> <td class="border-dark-blue table-bordered">$15.00</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">2 GB</th> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered">$23.00</td> <td class="border-dark-blue table-bordered">$10.00</td> <td class="border-dark-blue table-bordered">$20.00</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">3 GB</th> <td class="border-dark-blue border-left-0 border-right-0">$25.00</td> <td class="border-dark-blue border-left-0 border-right-0">$20.00</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$14.99</td> <td class="border-dark-blue table-bordered">$30.00</td> <td class="border-dark-blue table-bordered">N/A</td> <td class="border-dark-blue table-bordered">N/A</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">5 GB</th> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">N/A</td> <td class="border-dark-blue table-bordered">$28.00</td> <td class="border-dark-blue table-bordered">$21.21</td> <td class="border-dark-blue table-bordered">$25.00</td> </tr> <tr> <th class="table-dark-blue border-dark-blue table-sticky-column text-white table-bordered" scope="row">Unlimited</th> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$34.00</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$34.00</td> <td class="border-dark-blue table-bordered border-left-0 border-right-0">$29.99</td> <td class="border-dark-blue table-bordered">$50.00</td> <td class="border-dark-blue table-bordered">N/A</td> <td class="border-dark-blue table-bordered">$34.99</td> </tr> </tbody></table>
```

Comment:
@machikoyasuda Do you want to create a ticket for researching / spiking / refactoring this in the future? (as you mentioned in https://github.com/cal-itp/mobility-marketplace/pull/406#discussion_r1088214359)

Reply:
Done: #413